cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1459,19 +1459,21 @@
       "Condition": "CreateComputeFleet"
     },
     "SNS": {
-      "Type": "AWS::SNS::Topic",
+      "Type": "AWS::SNS::Topic"
+    },
+    "SNSSubscription": {
+      "Type": "AWS::SNS::Subscription",
       "Properties": {
-        "Subscription": [
-          {
-            "Endpoint": {
-              "Fn::GetAtt": [
-                "SQS",
-                "Arn"
-              ]
-            },
-            "Protocol": "sqs"
-          }
-        ]
+        "Endpoint": {
+          "Fn::GetAtt": [
+            "SQS",
+            "Arn"
+          ]
+        },
+        "Protocol": "sqs",
+        "TopicArn": {
+          "Ref": "SNS"
+        }
       },
       "Condition": "CreateComputeFleet"
```

Comment:
Do we need this condition for "SNS" as well?

Reply:
I think we don't need an SNSSubscription nor an SNS topic with awsbatch and slurm, which makes me think yes we need this condition and we need to also apply it to the SNS above

Reply:
Done